### ISSUE TITLE
D3G

### ISSUE THREAD

**Issue by erikhenicke** [erikhenicke/bigpicture#22]
- Train LR-domain-task specific and then introduce the other losses
- Use softmax(domain_logits) instead of argmax for D3G
- Explore hyperparameter settings
- Let D3G run with coordinates
- Regress coordinates from LR features
- Contrastive geographic objective (InfoNCE)
